housing_cooperative_base/wizard/rental_state_year_report_wizard.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class RentalStateYearReportWizard(models.TransientModel):
+    _name = "hc.rental.state.year.report.wizard"
+    _description = "Create Rental State Year Report"
+
+    name = fields.Char(compute="_compute_name", store=True)
+    year = fields.Integer(
+        string="Year",
+        required=True,
+        default=fields.Date.today().strftime("%Y"),
```

Comment:
Why not simply= `fields.Date.today().year`?

Reply:
Because I didn't know you could do that..